docs/introduction/faq-and-common-issues.md

```diff
@@ -22,7 +22,8 @@ For more detailed information about the Shopsys Framework, please see [Shopsys F
 - [How to change the behavior of the product search on the front-end?](#how-to-change-the-behavior-of-the-product-search-on-the-front-end)
 - [Why are e-mails not sent immediately but at the end of the script](#why-are-e-mails-sent-before-end-of-the-script-and-not-immediately)
 - [Where does the business logic belong?](#where-does-the-business-logic-belong)
-- [How can I create a friendly URL or Breadcrumb navigation for my entity?](#how-can-i-create-a-friendly-url-or-breadcrumb-navigation-for-my-entity)
+- [How can I create a friendly URL for my entity?](#how-can-i-create-a-friendly-url-for-my-entity)
+- [How can I create Front-end Breadcrumb navigation for my entity?](#how-can-i-create-front-end-breadcrumb-navigation-for-my-entity)
```

Comment:
Breadcrumb navigation is never created for an entity, always for a page (or a route) or a couple of pages (or routes).

Reply:
Changed